### DISCUSSION TITLE
max_sell_power - Unknown / Unavailable

### DISCUSSION THREAD

**Discussion by Roving-Ronin** [slipx06/sunsynk-power-flow-card#488] [Huawei / iStore]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi @slipx06 
Just wondering if max_sell_power might be able to support 'Unknown' and 'Unavailable' ?
Looking at it and the Huawei / iStore inverters sensor for this takes ~15min to update, probably given as its not a regularly changing sensor unlike other sensors. Thinking that instead of the card showing "Limit: 0" in these situations, that could give the wrong view to a user (that grid export is totally disabled) that it might be better to show the 'Unknown' and 'Unavailable'  status?

### Expected behaviour

'Unknown' and 'Unavailable' readings from sensors used to populate max_sell_power are passed through on the card, not just numerical values.

### Possible Solutions

See above.

### Mode

Both

### Context / Reason

Currently 'Unknown' and 'Unavailable'  is shows as "Limit: 0", that could be mistaken by a user to mean their inverter(s) have disabled all grid exports.

**Comment by slipx06** [comment]
Hi @Roving-Ronin. In addition to `max_sell_power:` there is a small icon on the card that shows you whether export is disabled or not so I don't think this is necessary. You can provide the following sensor to activate

`solar_sell_247: | Optional | switch.sunsynk_toggle_solar_sell | Displays icons to indicate if sell solar is active or not. The switch can be toggled by clicking on the icon (on/off, 1/0)`

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/11c1b7a5-7333-4b87-b1c5-22a0a767e8b8)
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/00154529-4c49-40e5-b301-2d6fc237f227)
